kythe/docs/schema/schema.txt

```diff
@@ -1201,6 +1201,33 @@ public class E {
 }
 --------------------------------------------------------------------------------
 
+[[refid]]
+ref/id
+~~~~~~~~~~~
+Brief description::
+  A *ref/name* B if A does not *ref* or *define* B but A's contents are
```

Comment:
If the edge is "ref/id", then A does "ref" B in some form. Also "contents" reads a bit weird, but I assume you mean "the bytes of the anchor span"?

Reply:
Changed the wording a bit.